### PR TITLE
[react-select] Fix a spelling error.

### PR DESCRIPTION
**Pull request by Losses**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/react-select/lib/components/Option.d.ts
Comment (claasahl) [inline types/react-select/lib/components/Option.d.ts]:
```suggestion
  /** Whether the option is focused. */
```

**Review by claasahl — CHANGES_REQUESTED** [review]
Hi @Losses 
Thanks for catching this.

I think the correct spelling is "Whether".

**Comment by Losses** [comment]
@claasahl oops :P

**Comment by Losses** [comment]
@typescript-bot All request has been resolved :P

**Comment by claasahl** [comment]
> 
> 
> @typescript-bot All request has been resolved :P

Hi @Losses 

I believe that you would have had to accept my suggested changes, otherwise @typescript-bot won't realize that progress had been made.
Would you like to create a new PR this?

**Comment by Losses** [comment]
@claasahl Okay, I will create one later :D

**Comment by amcasey** [comment]
I think the bot was waiting for an official sign-off or a LGTM comment.  I can re-open and merge if this is ready to go.

**Comment by claasahl** [comment]
Hi @amcasey 

This fix was addressed as part of PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33085

**Comment by amcasey** [comment]
@claasahl Thanks!

**Pull request closed without merging**
